client/scripts/views/pages/validators/substrate/presentation_component.ts

```diff
@@ -11,25 +11,29 @@ const PresentationComponent = (state, chain: Substrate) => {
     name: 'Current Validators',
     content: m('table.validators-table', [
       m('tr.validators-heading', [
-        m('th.val-name', 'Name'),
-        m('th.val-commission', 'Commission'),
         m('th.val-controller', 'Controller'),
         m('th.val-stash', 'Stash'),
-        m('th.val-total', 'Total Bonded'),
+        m('th.val-total', 'Total Stake'),
+        m('th.val-total', 'Own Stake'),
+        m('th.val-total', 'Other Stake'),
+        m('th.val-commission', 'Commission'),
+        m('th.val-points', 'Points'),
         // m('th.val-age', 'Validator Age'),
         m('th.val-action', ''),
       ]),
-      Object.keys(validators).filter((validator) => (validators[validator].isElected === true))
-        .sort((val1, val2) => validators[val2].exposure - validators[val1].exposure)
+      Object.keys(validators).filter((validator) => (
+        validators[validator].isElected === true && validators[validator].isWaiting === false
+      )).sort((val1, val2) => validators[val2].exposure - validators[val1].exposure)
         .map((validator) => {
-          const total = chain.chain.coins(validators[validator].exposure.total);
-          const bonded = chain.chain.coins(validators[validator].exposure.own);
-          const nominated = chain.chain.coins(total.asBN.sub(bonded.asBN));
+          const total = chain.chain.coins(validators[validator].exposure.total); // total stake
```

Comment:
lets use comments above variables, instead of inline comments.

Reply:
Yes, Agree